editor/src/core/webpack-loaders/file-loader.ts

```diff
@@ -0,0 +1,15 @@
+import { ProjectFile } from '../shared/project-file-types'
+import { LoadModule, MatchFile, ModuleLoader } from './loader-types'
+
+const matchFile: MatchFile = (filename: string) => {
+  return [/\.avif$/, /\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/].some((regxp) => regxp.test(filename))
```

Comment:
Since these always appear to be suffixes you might want to make these straight strings and use `.endsWith` instead as regexes are sloooooow.

Reply:
Good shout

Reply:
Fixed in 828a4a3